# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../prozilla/gcc41.patch
# Copyright (C) 2006 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

diff -Nur prozilla-2.0.4-orig/src/download_win.h prozilla-2.0.4/src/download_win.h
--- prozilla-2.0.4-orig/src/download_win.h	2006-02-08 17:18:59.000000000 +0000
+++ prozilla-2.0.4/src/download_win.h	2006-04-23 18:38:08.229462500 +0000
@@ -52,7 +52,7 @@
   void handle_joining_thread();
   void handle_dl_fatal_error();
   void cleanup(boolean erase_dlparts);
-  void DL_Window::print_status(download_t * download, int quiet_mode);
+  void print_status(download_t * download, int quiet_mode);
 
   connection_t *connection;
   download_t *download;
